config/betamocks/services_config.yml

```diff
@@ -926,3 +926,11 @@
       :cache_multiple_responses:
         :uid_location: url
         :uid_locator:  '\/avs\/(.*)'
+
+# Simple Token Service API
+- :name: "STS"
+  :base_uri: <%= "#{URI(Settings.mobile_application_platform.oauth_url).host}:#{URI(Settings.mobile_application_platform.oauth_url).port}" %>
+  :endpoints:
+    - :method: :post
+      :path: "/sts/oauth/v1/token"
+      :file_path: "sts/token"
```

Comment:
Can we make the file path `map/sts/token`?

Reply:
Sure, just updated the mock data directory as well.